spec/hydra/pcdm/services/collection/remove_collection_spec.rb

```diff
@@ -23,17 +23,21 @@
       end
 
       it 'should remove collection while changes are in memory' do
+      skip( "pending resolution of AF-agg 52") do
         expect( Hydra::PCDM::RemoveCollectionFromCollection.call( subject, collection1 ) ).to eq collection1
         expect( Hydra::PCDM::GetCollectionsFromCollection.call( subject )).to eq []
       end
+      end
 
       it 'should remove collection only when objects and all changes are in memory' do
+      skip( "pending resolution of AF-agg 52") do
```

Comment:
This is resolved: https://github.com/projecthydra-labs/activefedora-aggregation/issues/52